sencha-workspace/SlateDemonstrationsStudent/app/view/main/Main.js

```diff
@@ -12,12 +12,11 @@ Ext.define('SlateDemonstrationsStudent.view.main.Main', {
     requires: [
         'Ext.window.MessageBox',
 
-        'SlateDemonstrationsStudent.view.main.MainController',
+        'SlateDemonstrationsStudent.controller.MainController',
```

Comment:
@aliyeysides regular controllers don't get required by a view, they get required by the app via the `controllers` config